packages/lib-classifier/src/store/markings/Line.js

```diff
@@ -0,0 +1,100 @@
+import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
+import { autorun } from 'mobx'
+import { filter, skip, skipUntil } from 'rxjs/operators'
+
+import Mark from './Mark'
+
+const LineModel = types
+  .model('LineModel', {
```

Comment:
These models should reflect the drawing tools currently in use, so for the line tool, a mark snapshot should look something like:
```js
{
  "tool": 3,
  "frame": 0,
  "x1": 309.7053527832031,
  "y1": 326.549072265625,
  "x2": 764.7003173828125,
  "y2": 236.323974609375
}
```